app/src/panels/pie-chart/preview.svg

```diff
@@ -0,0 +1,29 @@
+<svg width="156" height="96" viewBox="0 0 156 96" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <g clip-path="url(#clip0_1280_167)">
+        <g filter="url(#filter0_d_1280_167)">
+            <rect x="18" y="10" width="120" height="76" rx="6" fill="var(--card-face-color)" class="glow" />
+            <rect x="19" y="11" width="118" height="74" rx="5" stroke="var(--theme--primary)"
+                stroke-width="2" />
+        </g>
+        <rect x="88" y="39" width="24" height="4" rx="1" fill="var(--theme--primary-subdued)" />
+        <rect x="88" y="46" width="41" height="4" rx="1" fill="var(--theme--primary-subdued)" />
+        <rect x="88" y="53" width="26" height="4" rx="1" fill="var(--theme--primary-subdued)" />
+        <path
+            d="M81 48C81 42.8403 79.4035 37.807 76.4296 33.5906C73.4557 29.3742 69.2501 26.1813 64.3896 24.4498C59.5292 22.7182 54.252 22.533 49.2821 23.9195C44.3122 25.306 39.8931 28.1962 36.6309 32.1938L56 48H81Z"
+            fill="color-mix(in srgb, var(--theme--background), var(--theme--primary) 75%)" />
```

Comment:
@rijkvanzanten Same thing as other, there is no 75% of primary in your theming, thoughts?